### PR TITLE
Indexable metadata

### LINKED ISSUE #26: Include `indexable_metadata` in the specification
In `bep001-naming-conventions.md` we have a 'meta'-key (belonging to a value), defined as [_indexable_metadata-<index>]. This is not ok since it hat an underscore (`_`). We should replace it with `fa`, `inv`, `echo` and `tsl`.

### PR DESCRIPTION
**Pull request by agahkarakuzu**
Related to #26. 

Edited by @KirstieJane: Closes #26 (I think! Correct me if I'm wrong!)

### CODE REVIEW

**Review by Gilles86 — APPROVED** [review]
Looks good to me.

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -247,14 +247,25 @@ Please note that the order of the `index` and the value of the associated
...
 
 | Allowed key tags | Value list | Associated metadata field |
 |---------|------------|---------------------|
+| echo    | 1,2,... N  | EchoTime            |
 | fa      | 1,2,... N  | FlipAngle           |
 | inv     | 1,2,... N  | InversionTime       |
-| echo    | 1,2,... N  | EchoTime            |
 | tsl     | 1,2,... N  | SpinLockTime        |
 
+Please note that `<indexable_metadata>-<index>` is not free form. Updates to the
```
Comment (KirstieJane) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:266]: :100:

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 2]
```diff
@@ -247,14 +247,25 @@ Please note that the order of the `index` and the value of the associated
 metadata field do NOT have to be coherent (i.e. `fa-1`,`fa-2` and `fa-3` can
 correspond to the `FlipAngle` of `35`, `10` and `25` degrees).
 
+If a filename contains more than one indexable metadata, included key tags MUST
```
Comment (KirstieJane) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:250]: Looks great!

**Review by KirstieJane — APPROVED** [review]
Approved! If @Gilles86 and I both think it looks good then I think we can merge away.... @agahkarakuzu do you want to press the big green button 😉

**Comment by agahkarakuzu** [comment]
@KirstieJane thank you for the review and the green button offer 😍

**Pull request merged**